cognite/client/data_classes/contextualization.py

```diff
@@ -1036,3 +1038,213 @@ def __init__(
 
 class ResourceReferenceList(CogniteResourceList[ResourceReference]):
     _RESOURCE = ResourceReference
+
+
+class DirectionWeights(CogniteObject):
+    """Direction weights for the text graph that control how far text boxes can be one from another
+    in a particular direction before they are no longer connected in the same graph. Lower value means
+    larger distance is allowed.
+
+    Args:
+        left (float | None): explanation
+        right (float | None): explanation
+        up (float | None): explanation
+        down (float | None): explanation
+    """
+
+    def __init__(
+        self,
+        left: float | None = None,
+        right: float | None = None,
+        up: float | None = None,
+        down: float | None = None,
+    ) -> None:
+        self.left = left
+        self.right = right
+        self.up = up
+        self.down = down
+
+    @classmethod
+    def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None = None) -> Self:
+        return cls(
+            left=resource.get("left", None),
+            right=resource.get("right", None),
+            up=resource.get("up", None),
+            down=resource.get("down", None),
+        )
+
+
+class CustomizeFuzziness(CogniteObject):
```

Comment:
It's a bit of an unfortunate name, straight from the context api.

Reply:
I thought about changing the name and I believe it's better to keep it consistent with the API for the sake of user experience. Now the name matches the API field exactly so it's easy to look it up in the docs and code.

The name really should changed on the API side which talked to Ola about. He doesn't like the name either.

Reply:
Ah, I see. This still has to be passed as `customize_fuzziness=CustomizeFuzziness(...)` so I think you are free to choose a better name for it 😄 

Maybe @erlendvollset thinks otherwise